postgres/init/21-portal-basedata.sql

```diff
@@ -40,6 +40,10 @@ COPY portal.company_status (company_status_id, label) FROM stdin;
 4	INACTIVE
 \.
 
+COPY portal.company_user_status (company_user_status_id, label) FROM stdin;
+1	INVITED
```

Comment:
I suggest we take 'ACTIVE' instead of 'INVITED" and we add 'INACTIVE"

Reply:
ok, I'll change that

Reply:
done, we now have ACTIVE and INACTIVE